src/com/amazon/ion/benchmark/Main.java

```diff
@@ -313,9 +316,18 @@ public static void main(String[] args) {
         if (optionsMap.get("--help").equals(true)) {
             printHelpAndExit();
         }
+       
         try {
-            OptionsMatrixBase options = OptionsMatrixBase.from(optionsMap);
-            options.executeBenchmark();
+        	 if (optionsMap.get("generate").equals(true)) {
+        		 try {
+        				WriteRandomIonValues.writeRandomDecimals(Integer.valueOf(optionsMap.get("<size>").toString()),optionsMap.get("<file>").toString());
+        			} catch (Exception e) {
+        				e.printStackTrace();
+        			}
+        	 }else {
+        		 OptionsMatrixBase options = OptionsMatrixBase.from(optionsMap);
+               options.executeBenchmark();
+			}
```

Comment:
There are tabs mixed with spaces in this code block, creating weird-looking indentation. This project uses spaces only for indentation (4 spaces per indent), so let's follow that in these changes. You can configure your IDE to insert four spaces when you press the TAB button.